### PR TITLE
Nodeset compiler with guid nodeids

### PR DESCRIPTION
**Pull request by paddor**
Refers open62541/open62541-nodeset-loader#253

This adds support for GUID Node IDs in nodeset compiler.

### CODE REVIEW

**Comment by paddor** [comment]
This is not quite correct yet. Leading zeroes from the int->hex_string conversion are missing.

**Comment by jpfr** [comment]
Cool! Please remove the [WIP] when you are happy.

File: tools/nodeset_compiler/backend_open62541_datatypes.py [hunk 1]
```diff
@@ -99,6 +99,8 @@ def generateNodeIdCode(value):
     elif value.s != None:
         v = makeCLiteral(value.s)
         return u"UA_NODEID_STRING(ns[%s], \"%s\")" % (value.ns, v)
+    elif value.g != None:
+        return u"UA_NODEID_GUID(ns[%s], %s)" % (value.ns, generateGuidCode(value.gAsStrings()))
     raise Exception(str(value) + " no NodeID generation for bytestring and guid..")
```
Comment (basyskom-meerkoetter) [inline tools/nodeset_compiler/backend_open62541_datatypes.py:104]: the error string in the exception still needs to be adjusted?

Comment (paddor) [reply tools/nodeset_compiler/backend_open62541_datatypes.py]: You're right. But that's not all. See above. My python skills are bad.

**Comment by paddor** [comment]
@jpfr This is ready for review.

**Pull request merged**